terraform/modules/front_door/waf.tf

```diff
@@ -152,4 +157,97 @@ resource "aws_wafv2_web_acl" "this" {
       sampled_requests_enabled   = true
     }
   }
+
+  rule {
+    name     = "login-ip-rate-limit"
+    priority = 7
+
+    action {
+      block {
+        custom_response {
+          response_code = 429
+        }
+      }
+    }
+
+    statement {
+      rate_based_statement {
+        aggregate_key_type = "IP"
+        limit              = 100

```

Comment:
This login IP rate limit is something Delta had and it looked reasonable to me. It's equivalent to 20 requests/min, which I think any IP doing above that is probably behaving badly